ska_sync/ska_sync_config

```diff
@@ -42,6 +43,7 @@ file_sync:
     - archive/aca_dark
     - archive/starcheck/starcheck.db3
     - archive/obspar/archfiles.db3
+    - archive/ocat_target_table.h5
```

Comment:
Probably just needs rebase but...

Reply:
I had a bit of deja vu on that. Anyway, that commit is gone now.